File: MODUL-6/SOAL 4/PRAK604-2410817120017-RIKAFAULIANARAHMI.py
baris_pertama = input() 
baris_kedua = input() 
hasil = "" 
jumlah_bintang = 0 
jumlah_pagar = 0 

if len(baris_pertama) == len(baris_kedua):
    for i in range(len(baris_pertama)):
        if baris_pertama[i] == ' ' and baris_kedua[i] == ' ':
            hasil +=" "
        elif baris_pertama[i] == baris_kedua[i]:
            hasil +="*"
            jumlah_bintang += 1
        else:
            hasil +="#"
            jumlah_pagar += 1
    print(hasil)
    print(f"* = {jumlah_bintang}")
    print(f"# = {jumlah_pagar}")
    if (jumlah_bintang >= jumlah_pagar):
        print("Pesan Asli")
    else:
        print("Pesan Palsu")

else:
    print("Panjang Kalimat berbeda, pesan palsu")

File: MODUL-6/SOAL 4/PRAK604-2410817120017-RIKAFAULIANARAHMI.c
#include <stdio.h> 
#include <string.h> 
 
int main () { 
    char baris_pertama[50]; 
    char baris_kedua[50]; 
    int jumlah_bintang = 0; 
    int jumlah_pagar = 0; 
 
    scanf(" %[^\n]%*c", &baris_pertama); 
    scanf(" %[^\n]%*c", &baris_kedua); 
    if (strlen(baris_pertama) == strlen(baris_kedua)){ 
        for (int i = 0; i < strlen(baris_pertama); i++) { 
            if (baris_pertama[i] == ' ' && baris_kedua[i] == ' ' ) { 
                printf(" "); 
            } 
            else if (baris_pertama[i] == baris_kedua[i]) { 
                printf("*"); 
                jumlah_bintang++; 
            } 
            else { 
                printf("#"); 
                jumlah_pagar++; 
            } 
        } 
        printf("\n* = %d", jumlah_bintang); 
        printf("\n# = %d", jumlah_pagar); 
        if (jumlah_bintang >= jumlah_pagar) { 
            printf("\nPesan Asli"); 
        } 
        else { 
            printf("\nPesan Palsu"); 
        } 
    } 
    else { 
        printf("Panjang Kalimat berbeda, pesan palsu"); 
    } 
 
    return 0; 
} 